torch/_dynamo/convert_frame.py

```diff
@@ -866,6 +866,21 @@ def count_args(code: CodeType) -> int:
 
     chromium_event_log.reset()
     chromium_start_time = time.time_ns()
+    if ca_metrics := torch._dynamo.utils.get_compiled_autograd_metrics(
+        locals.get("self")
+    ):
+        ca_id, ca_start_ns, ca_end_ns = ca_metrics
+        ca_event = f"compiled_autograd_{ca_id}"
```

Comment:
I don't know of any realy problem it would cause, but curious why you did it this way. So far the event names have been static strings.

Reply:
im moving it to the metadata dict, i just want to be able to refer to which graph corresponded to this compile event